polling_stations/apps/data_finder/views.py

```diff
@@ -143,6 +159,26 @@ def get_directions(self):
             )
         return None
 
+    def has_data_for_upcoming_election(self, loc):
+        council = self.get_council(loc)
+        try:
+            latest_import_event = council.latest_data_event(DataEventType.IMPORT)
+            elections = latest_import_event.election_dates
+            latest_election_date = max(elections)
+            if latest_election_date >= timezone.now().date():
+                return True
+        except DataEvent.DoesNotExist:
+            # Do we want this to be the default?
+            return False
+        except AttributeError as err:
+            if err.args[0] == "'NoneType' object has no attribute 'election_dates'":
+                return False
+            raise
```

Comment:
These exceptions also mean we're only showing data when there's an `IMPORT` type `DataEvent` object with the `election_dates` field populated.
Is this correct?